sigilrecorder.lic

```diff
@@ -0,0 +1,205 @@
+custom_require.call(%w[common common-arcana common-items common-travel drinfomon events spellmonitor equipmanager])
+
+class SigilRecorder
+  include DRC
+  include DRCA
+  include DRCI
+  include DRCT
+
+	def initialize
+		require 'yaml'
+		@data = YAML.load(File.read("./scripts/data/base-sigils.yaml"))
```

Comment:
The path won't work like this, because the script will live in scripts/ already. Change to data/base-sigils.yaml?

Reply:
Try `get_data('sigils')`

Reply:
I was seeing weirdness when using get_data and then trying to write to the file, but I think I might've been using the wrong syntax to write to the file when paired with get_data. I'll look a few things up and see if I can get it working with get_data instead